development/custom-image/install-deps-debian.sh

```diff
@@ -102,6 +102,19 @@ sudo apt-cache madison google-fluentd
 sudo apt-get install -y 'google-fluentd=1.*'
 sudo apt-get install -y google-fluentd-catch-all-config
 
+# install go
+ export GO_VERSION=1.18.2
+
+ export GOPATH=/workspace/go
+ export PATH=$GOPATH/bin:/usr/local/go/bin:$PATH
```

Comment:
Image creation creates new VM based on Debian image, runs this script, stops the VM and creates new image from the VM. Exporting variables won't survive VM shutdown.

Reply:
Exports removed